javaslang/src/test/java/javaslang/collection/HashMultimapTest.java

```diff
@@ -287,7 +287,7 @@ public void shouldCreateSortedMapFrom10Pairs() {
 
     @Test
     public void shouldNarrowMap() {
-        final HashMultimap<Integer, Number> int2doubleMap = this.<Integer, Number> emptyMap().put(1, 1.0d);
+        final HashMultimap<Integer, Number> int2doubleMap = (HashMultimap<Integer, Number>) this.<Integer, Number> emptyMap().put(1, 1.0d);
```

Comment:
I would not provide a default implementation for `AbstractMultimapTest.emptyMap()` because now we have to cast every `emptyMap()` result.
Should not be a big problem, we have only three impls in our test classes.